### PR TITLE
Mobile web 2021

### PR DESCRIPTION
**Pull request by fellowhuman1101**
Progress on #2151 

Staging URL: https://20211116t161440-dot-webalmanac.uk.r.appspot.com/en/2021/mobile-web
Featured stats: https://20211116t161440-dot-webalmanac.uk.r.appspot.com/en/2021/?feat=mobile-web

### CODE REVIEW

**Comment by rviscomi** [comment]
Resolved that merge conflict

File: src/content/en/2021/mobile-web.md [hunk 1]
```diff
@@ -1,20 +1,1150 @@
...
-featured_stat_label_3: TODO
+fellowhuman1101_bio: Jamie Indigo isn’t a robot, but speaks bot.  As a technical SEO consultant at <a href="https://www.deepcrawl.com">Deepcrawl</a>, they study how search engines crawl, render, and index the web.  They love to tame wild JavaScript frameworks and optimize rendering strategies. When not working, Jamie likes horror movies, graphic novels, and Dungeons & Dragons.
+dwsmart_bio: Dave Smart is a developer and technical search engine consultant at <a href="https://tamthebots.com">Tame the Bots</a>. They love building tools and experimenting with the modern web, and can often be found at the front in a gig or two.
+featured_quote: In 2021, the perception of a distinct "mobile web" is outdated. Across multiple data sources, it seems that the mobile is one of many ways a user can interact with digital content.
+featured_stat_1: 18.4%
+featured_stat_label_1: Mobile page loads using native lazy-loading
+featured_stat_2: 43.4%
+featured_stat_label_2: Mobile page loads contain inappropriately sized images
+featured_stat_3: 45.03%
+featured_stat_label_3: Of the top 1000 mobile page loads prevent zooming
```
Comment (rviscomi) [inline src/content/en/2021/mobile-web.md:19]:
```suggestion
featured_stat_label_3: Of the top 1000 mobile page loads prevent zooming
unedited: true
```

File: src/content/en/2021/mobile-web.md [hunk 2]
```diff
@@ -1,20 +1,1150 @@
...
+   caption="Usage of Network Client Hint Directives",
+   description="Bar chart showing the usage of network client hint directives detected on mobile and desktop page loads",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQv2xPBvpLK9_1QqEmOhsXHXcCzunEdsY14Xzeo1D7MYTuu0inzwmz0NtGSFI0mBRP5snPw8ciVWaJQ/pubchart?oid=172140786&format=interactive",
+   sheets_gid="1041308066",
+  sql_file="client_hints.sql"
+   )
+}}
+
+
+Network client hints show a similar uptake to Device client hints, with [Downlink](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Downlink)and [ECT ](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ECT)(effective connection type) being requested by **0.15%** of loads on mobile, and [RTT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/RTT) (round trip time) on 0.14% of loads on mobile.
```
Comment (rviscomi) [inline src/content/en/2021/mobile-web.md:251]:
```suggestion
Network client hints show a similar uptake to Device client hints, with [Downlink](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Downlink)and [ECT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ECT) (effective connection type) being requested by **0.15%** of loads on mobile, and [RTT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/RTT) (round trip time) on 0.14% of loads on mobile.
```

**Review by rviscomi — APPROVED** [review]
This LGTM to move on to final editing.

@obto @tunetheweb any feedback before we merge?

**Comment by tunetheweb** [comment]
Only had a brief glance previously, and not read it through, but was looking good from a quick skim at the markup so agree.

Can we change it back to TODO though for the pre launch?

File: src/config/2021.json [hunk 3]
```diff
@@ -98,8 +98,7 @@
           "part": "II",
           "chapter_number": "12",
           "title": "Mobile Web",
-          "slug": "mobile-web",
-          "todo": "true"
+          "slug": "mobile-web"
```
Comment (rviscomi) [inline src/config/2021.json:101]:
```suggestion
          "slug": "mobile-web"
          "todo": "true"
```

**Comment by tunetheweb** [comment]
Can I also say great job on the conversion. Think this is the most hands off I’ve been yet on a chapter conversion so well done @dwsmart and @fellowhuman1101 !

**Comment by dwsmart** [comment]
Thanks @tunetheweb!

**Comment by obto** [comment]
Sorry I couldn't comment in time. Thanks so much for carrying this to the finish line @dwsmart @fellowhuman1101

**Pull request merged**